app/src/main/res/values/strings.xml

```diff
@@ -256,4 +256,10 @@
     </string>
     <string name="empty_list">There are no codes to be shown. Start adding entries by tapping the plus sign in the bottom right corner</string>
     <string name="empty_list_title">No entries found</string>
+    <string name="done">Done</string>
+    <string name="entries_count">%d / %d entries</string>
+    <string name="next">Next</string>
+    <string name="previous">Previous</string>
+    <string name="transfer_entry">Transfer entry</string>
+    <string name="transfer_entry_description">Scan this QR code with your preferred authenticator app</string>
```

Comment:
I suggest rewording this to: Scan this QR code with the authenticator app you'd like to transfer this entry to.